composer.json

```diff
@@ -93,7 +95,8 @@
         "sort-packages": true,
         "allow-plugins": {
             "infection/extension-installer": true,
-            "composer/package-versions-deprecated": true
+            "composer/package-versions-deprecated": true,
+            "yiisoft/config": false
```

Comment:
?

Reply:
It was automatically after config installation